.gitignore

```diff
@@ -1,2 +1 @@
-test.md
-node_modules
\ No newline at end of file
+test.md
```

Comment:
- [ ] your gitignore file is supposed to be ignoring two files, `test.md` and `node_modules` however it is ignoring only the test.md please try adding the node_modules before making your next pull request. :+1: